### ISSUE TITLE
Add Kotlin example for SecuritySocketAcceptorInterceptor of RSocket

### ISSUE THREAD

**Issue by talerngpong** [spring-projects/spring-security#10932]
**Describe the bug**
An example of `Adding SecuritySocketAcceptorInterceptor` bullet comes with only Java example under RSocket Security.

**To Reproduce**
1. Browse to [Adding SecuritySocketAcceptorInterceptor](https://docs.spring.io/spring-security/reference/5.7.0-M2/reactive/integrations/rsocket.html#_adding_securitysocketacceptorinterceptor) bullet.
2. Visually compare the bullet with the previous one [Minimal RSocket Security Configuration](https://docs.spring.io/spring-security/reference/5.7.0-M2/reactive/integrations/rsocket.html#_minimal_rsocket_security_configuration).
3. A coding block of `Adding SecuritySocketAcceptorInterceptor` comes with one Java example while a coding block of `Minimal RSocket Security Configuration` comes with both of Java and Kotlin examples.

**Expected behavior**
A coding block of `Adding SecuritySocketAcceptorInterceptor` should come with both of Java and Kotlin examples.

**Sample**
```java
// current example
@Bean
RSocketServerCustomizer springSecurityRSocketSecurity(SecuritySocketAcceptorInterceptor interceptor) {
    return (server) -> server.interceptors((registry) -> registry.forSocketAcceptor(interceptor));
}
```
```kotlin
// purposed example to be added
@Bean
fun springSecurityRSocketSecurity(interceptor: SecuritySocketAcceptorInterceptor): RSocketServerCustomizer {
    return RSocketServerCustomizer { server ->
        server.interceptors { registry ->
            registry.forSocketAcceptor(interceptor)
        }
    }
}
```

**Comment by eleftherias** [comment]
Thank you for submitting this issue @talerngpong! I will review the associated PR #10936 today.

**Comment by eleftherias** [comment]
Closed via [bc2bb8c](https://github.com/spring-projects/spring-security/commit/bc2bb8cb9659c02ccedd440156943d9849c63550).
